### PR TITLE
Added greek layout

### PR DESCRIPTION
**Pull request by iByteABit256**
I used [this](https://www.researchgate.net/publication/220469197_Basic_Quantitative_Characteristics_of_the_Modern_Greek_Language_Using_the_Hellenic_National_Corpus) paper to determine the letter frequency for Greek which is based on the Hellenic National Corpus containing more than 48,000 written texts totaling 33 million words.

### CODE REVIEW

**Comment by iByteABit256** [comment]
### Without shift

Screenshot_2023-08-21-23-43-32-733_org thoughtcrime securesms

### With shift

Screenshot_2023-08-21-23-41-33-124_org thoughtcrime securesms

**Comment by iByteABit256** [comment]
## Without shift

Screenshot_2023-08-21-23-45-53-513_org thoughtcrime securesms

---

## With shift
Screenshot_2023-08-21-23-41-33-124_org thoughtcrime securesms

File: app/src/main/java/com/dessalines/thumbkey/utils/Types.kt [hunk 1]
```diff
@@ -104,6 +104,7 @@ enum class KeyboardLayout(val title: String, val index: Int) {
     ThumbKeyFIv1("Thumb-Key suomi v1", 7),
     ThumbKeyFIv1Wide("Thumb-Key suomi v1 wide", 22),
     ThumbKeyFRv1("Thumb-Key français v1", 8),
+    ThumbKeyGRv1("Thumb-Key ελληνικά v1", 51),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/Types.kt:107]: Add this at the bottom of the list.

**Review by dessalines — CHANGES_REQUESTED** [review]

**Review by dessalines — APPROVED** [review]
Perfect, thanks!

**Pull request merged**